### PR TITLE
Re-adds content for setupDeviceLock

### PR DESCRIPTION
**Pull request by lysannep**
### Description
Restores content for setupDeviceLock (https://docs.saucelabs.com/dev/test-configuration-options/#setupdevicelock)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/test-configuration-options.md [hunk 1]
```diff
@@ -774,6 +774,22 @@ capabilities.setCapability("sauce:options", sauceOptions);
 
 ---
 
+### `setupDeviceLock`
+<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
```
Comment (wswebcreation) [inline docs/dev/test-configuration-options.md:778]: Can we keep the values in line with all the other options?
```suggestion
<p><small>|OPTIONAL | BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
```

File: docs/dev/test-configuration-options.md [hunk 2]
```diff
@@ -774,6 +774,22 @@ capabilities.setCapability("sauce:options", sauceOptions);
...
+
+Sets up the device pin code for the automated test session. Valid values are `true` and `false`.
+This capability sets your device in the state required for your application to launch successfully.
+
+:::important
+The `setupDeviceLock` capability helps to bypass the Security requirements from your applications, like pincode requirements for launching and app or invoking certain activities/features within your app. For an example, see https://developer.android.com/reference/android/app/KeyguardManager.
+:::
+
+```java title="Real Device Setting"
+"setupDeviceLock": "true"
```
Comment (wswebcreation) [inline docs/dev/test-configuration-options.md:788]: Can we keep the code examples aligned with what we have for all RDC options?

```suggestion
MutableCapabilities capabilities = new MutableCapabilities();
//...
MutableCapabilities sauceOptions = new MutableCapabilities();
sauceOptions.setCapability("setupDeviceLock", true);
capabilities.setCapability("sauce:options", sauceOptions);
```

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Thanks for putting this back, can you please check the comments?

**Review by aoleszyn — APPROVED** [review]

**Comment by lysannep** [comment]
Thanks for your comments @wswebcreation!

**Pull request merged**